src/services/cleeng.account.service.ts

```diff
@@ -43,6 +48,31 @@ export const login: Login = async ({ config, email, password }) => {
   };
 };
 
+export const register: Register = async ({ config, email, password }) => {
+  const localesResponse = await getLocales(!!config.integrations.cleeng?.useSandbox);
+
+  if (localesResponse.errors.length > 0) throw new Error(localesResponse.errors[0]);
```

Comment:
I suggest making a helper method for this pattern since it's already used several times.

Reply:
Okay, makes sense.